Tests/SwiftFormatPrettyPrintTests/TernaryExprTests.swift

```diff
@@ -41,11 +41,12 @@ public class TernaryExprTests: PrettyPrintTestCase {
       """
     let expected =
       """
-      let someLocalizedText = shouldUseTheFirstOption
-        ? stringFunc(name: "Name1", comment: "short comment")
-        : stringFunc(
-          name: "Name2", comment: "Some very, extremely long comment",
-          details: "Another comment")
+      let someLocalizedText =
+        shouldUseTheFirstOption
```

Comment:
It seems like this should still fit on the previous line. Was this break a result of additional grouping?

Reply:
The input text has a newline after the `=`, so it was "invalid" before and got removed. Now it's being preserved as a discretionary line break.

Reply:
Doh. I should've seen that in the input. Thanks for clarifying.